api/v3/Order.php

```diff
@@ -59,3 +59,42 @@ function civicrm_api3_order_get($params) {
   $params['sequential'] = $isSequential;
   return civicrm_api3_create_success($contributions, $params, 'Order', 'get');
 }
+
+/**
+ * Delete a Order.
+ *
+ * @param array $params
+ *   Input parameters.
+ *
+ * @return array
+ */
+function civicrm_api3_order_delete($params) {
+  $contribution = civicrm_api3('Contribution', 'get', array(
```

Comment:
I had tried with 

  $result = civicrm_api3('Contribution', 'getvalue', array(
      'sequential' => 1,
      'return' => array("is_test"),
      'id' => 22,
    ));

I was getting exception error with 'field Array unset or not existing'.